recipe/meta.yaml

```diff
@@ -9,7 +9,7 @@ source:
   sha256: 13f93ce9bea8016c253b34afc6bd6a75993e5c40672ed5405a9c832f0d4a00bc
 
 build:
-  number: 0
+  number: 1
```

Comment:
By bumping the build number like this, we'll lose Python 3.12...

I think we could keep the build number to `0`, add back the abs file with the aggregate branch and change the skip to `# [py<37 or win32 or (py312 and ppc64le)]`. By default our builds will skip existing (on defaults) packages (using the `--skip-existing` flag).

Reply:
it doesn't need to be built for python 3.12 on ppc. There were some unsatisfiability errors that looked like a build of something else would be necessary and decided not to, see [here](https://anaconda.slack.com/archives/C03J3NQPS0Z/p1702325455587259?thread_ts=1702308712.552099&cid=C03J3NQPS0Z)

Reply:
@danpetry what I'm saying is that as it is right now, build 1 won't have any python 3.12 builds on the other platforms, which is a regression.

Reply:
Alright, maybe not a regression after all. But it creates an inconsistency (python 3.12 will only be available on build 0, which is annoying when it comes to debugging resolves). But it's not a blocker.

Reply:
Sorry I haven't been paying close attention. I'm also not sure why bumping the build number is required? I thought we would just build the ppc platform after this, not the others...